### DISCUSSION TITLE
Completing the Swagger -> OpenAPI rename

### DISCUSSION THREAD

**Discussion by RSO** [rswag/rswag#709] [General]
#574 started renaming Swagger -> OpenAPI, which led to a 2.x release that deprecates a bunch of fields. When refactoring my codebase, I was surprised to find out that the the renaming in 2.x was only partial. Namely:
- `rswag:specs:swaggerize` still uses the `swagger` term
- `swagger_helper.rb` is still an expected filename
- `/swagger` is (I think) still an expected location for the `swagger.yaml` file

So I guess I'm wondering: Are there places where we intend to keep using the term `swagger`, or should v3 bring along a complete ban on the word `swagger`?

**Comment by romanblanco** [comment]
@RSO, [v3](https://github.com/rswag/rswag/pull/714) pretty much abandons using "Swagger" and replaces it with "OpenAPI".

`swaggerize` task has not been changed yet, but there have been [suggestions](https://github.com/rswag/rswag/pull/574#issuecomment-1646314110) to also convert it and use an alias, so for consistency it might also be changed to `openapify` or something better.
